packages/photon/src/generation/generateClient.ts

```diff
@@ -295,3 +305,143 @@ async function fileSize(name: string): Promise<number | null> {
     return null
   }
 }
+
+function validateDmmfAgainstDenylists(prismaClientDmmf) {
+  const errorArray = [] as Error[]
+
+  const denylists = {
+    models: [
+      'Enumerable',
+      'MergeTruthyValues',
+      'CleanupNever',
+      'AtLeastOne',
+      'OnlyOne',
+      'StringFilter',
+      'IDFilter',
+      'FloatFilter',
+      'IntFilter',
+      'BooleanFilter',
+      'DateTimeFilter',
+      'NullableStringFilter',
+      'NullableIDFilter',
+      'NullableFloatFilter',
+      'NullableIntFilter',
+      'NullableBooleanFilter',
+      'NullableDateTimeFilter',
+      'PrismaClientFetcher',
+      'PrismaClient',
+      'Engine',
+      'PrismaClientOptions',
```

Comment:
Is this the list of fixed strings that are not allowed in model names? 
Is this exhaustive?

Reply:
Not allowed in models and enum names indeed. Not 100% sure that it is exhaustive, will need to check.

Reply:
Probably missing list: @timsuchanek will know
```
atMostOne
missingArg

// maybe?
ArgError

InvalidFieldError
InvalidFieldNameError
InvalidFieldTypeError
EmptySelectError
NoTrueSelectError
IncludeAndSelectError
EmptyIncludeError

InvalidArgError
InvalidArgNameError
MissingArgError
InvalidArgTypeError
AtLeastOneError
AtMostOneError
```

Reply:
What needs to go in here:
- The new errors like PrismaClientKnownRequestError
- the ts utility types

There might be some crazy deeply nested input types, that we're not covering right now, but that's fine for now. We e.g. sometimes have names like "CreateUserWithoutPostInput"